src/bgw/job.c

```diff
@@ -377,7 +372,7 @@ ts_bgw_job_get_scheduled(size_t alloc_size, MemoryContext mctx)
 
 #ifdef USE_TELEMETRY
```

Comment:
note: `ts_is_telemetry_job` is always compiled - no need to guard with `ifdef`

Reply:
Yes, but if compiling without USE_TELEMETRY, it is better to just remove all the code since it should not be present. I considered removing `ts_is_telemetry_job` as well.

Reply:
Removed the function entirely. It will not work if you try to call non-telemetry Apache code from telemetry-enabled TSL code, but that is not something we should support anyway.